### PR TITLE
For #11118: Add missing telemetry

### PR DESCRIPTION
**Pull request by sblatz**
This is mostly an extension of https://github.com/mozilla-mobile/fenix/pull/11211, just tying up a couple loose ends on data we missed that's important. Also we now have more confidence that this format works, so we're adding a couple more.

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt [hunk 1]
```diff
@@ -614,6 +662,17 @@ class GleanMetricsService(private val context: Context) : MetricsService {
...
+            val toolbarPositionSelection =
+                if (context.settings().shouldUseFixedTopToolbar) {
+                    "fixed_top"
+                } else if (context.settings().shouldUseBottomToolbar) {
+                    "bottom"
+                } else {
+                    "top"
+                }
+
+            toolbarPosition.set(listOf(toolbarPositionSelection))
```
Comment (liuche) [inline app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt:674]: Is this a new addition? If so, please document

File: app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt [hunk 2]
```diff
@@ -603,6 +649,8 @@ class GleanMetricsService(private val context: Context) : MetricsService {
             showSearchShortcuts.set(context.settings().shouldShowSearchShortcuts.toStringList())
             openLinksInAPrivateTab.set(context.settings().openLinksInAPrivateTab.toStringList())
             searchSuggestionsPrivate.set(context.settings().shouldShowSearchSuggestionsInPrivate.toStringList())
+            showVoiceSearch.set(context.settings().shouldShowVoiceSearch.toStringList())
```
Comment (liuche) [inline app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt:652]: Are these new additions to the pref-toggle-event or prefs-metrics ping? If so please document. Or was it one of the missing pings?

File: app/metrics.yaml [hunk 3]
```diff
@@ -728,6 +728,34 @@ preferences:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+  open_links_in_app:
+    type: string_list
+    description: >
+      Whether or not the user has the open links in apps feature enabled.
```
[the remark this answers is not in the document]
Comment (liuche) [reply app/metrics.yaml:734]: Okay pt 3, I see we don't do this for other booleans either. I think this should be pretty clear, so just ignore me 😂

File: app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt (hunk 2 above)
Comment (sblatz) [reply app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt:652]: Nope! Part of the initial telemetry I forgot.

File: app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt (hunk 1 above)
Comment (sblatz) [reply app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt:674]: Nope! Part of the initial telemetry I forgot.

**Pull request merged**